connector/src/main/scala/com/microsoft/kusto/spark/datasink/KustoWriter.scala

```diff
@@ -397,6 +398,42 @@ object KustoWriter {
     writer.out.toString
   }
 
+  private def writeDecimalField(row: SpecializedGetters, fieldIndexInRow: Int, precision: Int,scale: Int, writer: Writer) = {
+    writer.write('"')
```

Comment:
Although you are highly covered with E2E tests I would think it will be better to test the decimal to string representation using a simple unit tests. LMK WDYT

Reply:
The tests under WriterTests are not E2E . I added there a test in this PR already